pkg/cinderbackup/statefulset.go

```diff
@@ -63,24 +58,35 @@ func StatefulSet(
 	}
 
 	args := []string{"-c"}
+	var probeCommand []string
+	// When debugging the service container will run kolla_set_configs and
+	// sleep forever and the probe container will just sleep forever.
 	if instance.Spec.Debug.Service {
 		args = append(args, common.DebugCommand)
 		livenessProbe.Exec = &corev1.ExecAction{
 			Command: []string{
 				"/bin/true",
 			},
 		}
+		startupProbe.Exec = livenessProbe.Exec
+		probeCommand = []string{
+			"/bin/sleep", "infinity",
+		}
 	} else {
 		args = append(args, ServiceCommand)
-		livenessProbe.Exec = &corev1.ExecAction{
-			Command: []string{
-				"/usr/local/bin/container-scripts/healthcheck.sh",
-				"cinder-backup",
-			},
+		// Use the HTTP probe now that we have a simple server running
+		livenessProbe.HTTPGet = &corev1.HTTPGetAction{
+			Port: intstr.FromInt(8080),
+		}
+		startupProbe.HTTPGet = livenessProbe.HTTPGet
+		// Probe doesn't run kolla_set_configs, so cinder.conf is in its
```

Comment:
Any particular reason to not run kolla_set_config? It might be necessary to set the correct owner/mode if not running as root

Reply:
If we run `kolla_set_config` with a non-root user it will fail due to permissions, as it needs to set some configurations to be owned by `root`.